views/manage/invites.tt

```diff
@@ -0,0 +1,134 @@
+[%# manage/invites.tt
+  #
+  # Authors:
+  #     Jen Griffin <kareila@livejournal.com>
+  #
+  # Copyright (c) 2023 by Dreamwidth Studios, LLC.
+  #
+  # This program is free software; you may redistribute it and/or modify it under
+  # the same terms as Perl itself.  For a copy of the license, please reference
+  # 'perldoc perlartistic' or 'perldoc perlgpl'.
+  #
+%]
+
+[%- sections.title = '.title' | ml -%]
+
+[%- CALL dw.active_resource_group( "foundation" ) -%]
+
+[%- sections.head = BLOCK %]
+    <style type="text/css">
+        #invites-list {
+            width: 100%;
+        }
+        #invites-list td.invite-info {
+            padding-left: 22px;
+            text-indent: -18px;
+        }
+        .invite-from {
+            color: #777;
+        }
+        .invite-accept,
+        .invite-decline {
+            white-space: nowrap;
+        }
+        #content input[type="radio"] {
+            margin: 0;
+        }
+    </style>
+[%- END -%]
+
+[% IF responses %]
+  [% IF responses.accepted.size %]
+<h2>[% '.head.accepted' | ml %]</h2>
+<p>[% '.body.accepted' | ml %]</p>
+<dl>[% FOREACH row IN responses.accepted %]
+  <dd>[% row.0.ljuser_display %]:
+      [% FOREACH attr IN row.1;
+           IF loop.count != 1; ", "; END; ".label.$attr" | ml;
+         END %]</dd>
+    [% END %]
+</dl>
+<br />
+  [% END %]
+
+  [% IF responses.rejected.size %]
+<h2>[% '.head.rejected' | ml %]</h2>
+<p>[% '.body.rejected' | ml %]</p>
+<dl>[% FOREACH comm IN responses.rejected %]
```

Comment:
I know you're just pulling the `<dl>`/`<dd>` from the existing BML template, but I think this isn't really an appropriate use of them, and they can be weird to style -  I think this should probably be `<ul>`/`<li>` instead (possibly with `list-style: none;` CSS)

Reply:
OR ACTUALLY. If you put the comm name inside of `<dt>` tags, and the attributes list in a `<dd>` tag, that would be appropriate use of the tags, semantically speaking. But may change the visual look. (this assumes the attributes are in some way descriptions of the comm)